staging/src/k8s.io/apiextensions-apiserver/pkg/apiserver/customresource_handler.go

```diff
@@ -314,6 +316,10 @@ func (r *crdHandler) ServeHTTP(w http.ResponseWriter, req *http.Request) {
 		return
 	}
 
+	// kcp
+	req = utilnet.CloneRequest(req)
+	req = req.WithContext(kcpapi.WithCustomResourceIndicator(req.Context()))
```

Comment:
indicator?

Reply:
So way to indicate this is a request for a CR